C++/include/algorithms/string/splice.hpp

```diff
@@ -0,0 +1,48 @@
+/*
+	String Splice Function
+	----------------------
+	Breaks a given sentence into individual words and returns
+	a string.
+	The function takes two arguments as input a string and the
+	position of word you want to get in return.
+	For Example if you call:
+	splice("Hello I am Utkarsh", 4)
+	The returned value will be "Utkarsh".
+
+	Time complexity
+	---------------
+	O(n), where n is the length of the string.
+
+	Space complexity
+	----------------
+	O(n), where n is the length of the string.
+*/
+
+#include <iostream>
+#include <string.h>
+
+using namespace std;
```

Comment:
Don't use a namespace in a header file. This can lead to "namespace pollution" in the files that include this header.